app/controllers/api/v0/organizations_controller.rb

```diff
@@ -21,6 +21,14 @@ class OrganizationsController < ApiController
       ].freeze
       private_constant :LISTINGS_FOR_SERIALIZATION
 
+      ARTICLES_FOR_SERIALIZATION = %i[
+        id user_id organization_id collection_id
+        title description main_image published_at crossposted_at social_image
+        cached_tag_list slug path canonical_url comments_count
+        public_reactions_count created_at edited_at last_comment_at published
+        updated_at video_thumbnail_url
+      ].freeze
+
```

Comment:
I wonder if, instead copying `Api::V0::ArticlesController::INDEX_ATTRIBUTES_FOR_SERIALIZATION` here, we can't just refer to it in the controller, this way we make sure that the fields in both endpoints are always aligned.

Reply:
I updated this